src/databases/PLOT3D/vtkPLOT3DReader.C

```diff
@@ -235,8 +266,13 @@ int vtkPLOT3DReader::AutoDetectionCheck(FILE* fp)
     this->Internal->MultiGrid = this->MultiGrid;
     this->Internal->NumberOfDimensions = this->TwoDimensionalGeometry ? 2 : 3;
     this->Internal->Precision = this->DoublePrecision ? 8 : 4;
-    this->Internal->IBlanking = this->IBlanking;
+    this->Internal->IBlanking = this->IBlankingInFile;
     }
+
+  std::cout << "After Auto-Detection, IBlankingInFile is " << this->IBlankingInFile << std::endl;
```

Comment:
Remove this debugging information. It would probably be a good idea to write it to one of the debug logs.

Reply:
Oh, wait, this is a vtk function. Never mind.

Reply:
> Oh, wait, this is a vtk function. Never mind.

FWIW...I am not surre "...is a vtk function..." matters all that much for deciding whether debug log statements are allowed or a good idea. This is part of a reader's behavior and if it could help debugging, the more (debug statements) the merrier 😆 IMHO. But deciding which debug *level* is a good idea. Anything that would produce output that scales with `NumNodes` or `NumZones` should only go to level 5 logs (`debug5 << "After auto-detection...`).

Reply:
I just used this to make sure that iblanking was being detected and applied as expected.